code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -74,3 +74,9 @@
 		user << "<span class='notice'>..and falls into view. Whew, that was a close one.</span>"
 		user.unEquip(src)
 
+/obj/item/weapon/gun/projectile/automatic/pistol/mgs

```

Comment:
that'd be more reasonable, yeah, i'll do it.

IM SEARCHING AND IM MELTING INTO YOU

WHAT A FEAR IN MY HEART